project1/queries.py

```diff
@@ -93,15 +94,19 @@
 select 0;
 """
 
-### 8. Write a query to find customers that took flights over four consecutive days, but did not fly any other day.
+### 8. Write a query to find customers that took flights on four consecutive days, but did not fly any other day.
 ###    Return the name, start and end date of the customers flights.
 ### Output: customer_name, start_date, end_date
 ### Order: by customer_name
 queries[8] = """
 select 0;
 """
 
-### 9. Write a query to find all flights belonging to the same airline that had a layover in JFK between 1 and 3 hours in length (inclusive)
+### 9. A layover consists of set of two flights where the destination of the first flight is the same 
+###    as the source of the second flight. Additiobally, the arrival of the first flight must be before the
+###    departure of the first flight. 
+###    Write a query to find all flights belonging to the same airline that had a layover in JFK between
```

Comment:
"... to find all **pairs of flights** belonging to ..."

Reply:
Done.